### PR TITLE
Changing "add-on" language for Experimentation

### PR DESCRIPTION
**Pull request by cameronsavage**
We made a conscious pivot to position Experimentation as a Product equivalent to Feature Management, so dropping the "add-on" language while still calling out that Experimentation is only available for Pro and Ent

### CODE REVIEW

File: src/content/topics/home/experimentation-about/index.mdx [hunk 1]
```diff
@@ -7,10 +7,10 @@ tags: ['experimentation', 'experiment', 'ab testing', 'enterprise']
 ---
 
 <Callout intent="primary">
-<CalloutTitle>Experimentation is an add-on feature</CalloutTitle>
+<CalloutTitle>Available for Pro and Enterprise plans</CalloutTitle>
 <CalloutDescription>
 
-Experimentation is available as an add-on for customers on a Pro or Enterprise plan. To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To add Experimentation to your plan, [contact Sales](https://launchdarkly.com/contact-sales/).
+The Experimentation product is available to purchase for customers on a Pro or Enterprise plan. To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To add Experimentation to your plan, [contact Sales](https://launchdarkly.com/contact-sales/).
```
Comment (scribblingfox) [inline src/content/topics/home/experimentation-about/index.mdx:13]:
```suggestion
Experimentation is available to customers on a Pro or Enterprise plan. To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To add Experimentation to your plan, [contact Sales](https://launchdarkly.com/contact-sales/).
```

**Review by scribblingfox — CHANGES_REQUESTED** [review]

**Comment by scribblingfox** [comment]
We can totally drop the add-on language, but we don't refer to any of LaunchDarkly's other products as "products," including the LD platform itself. Is it cool if we strike that word?

**Comment by cameronsavage** [comment]
That works, 

> We can totally drop the add-on language, but we don't refer to any of LaunchDarkly's other products as "products," including the LD platform itself. Is it cool if we strike that word?

That works for me @scribblingfox

**Review by scribblingfox — APPROVED** [review]

**Comment by mmrj** [comment]
Once the text is finalized, we should make the equivalent change on the other pages where this callout appears:
* /guides/experimentation/* (there are five)
* /home/using-flags/targeting_users/targeting-rules

**Comment by mmrj** [comment]
> Once the text is finalized, we should make the equivalent change on the other pages where this callout appears

made a followup ticket in git-gatsby, this can be merged

**Pull request merged**